Testskript.R

```diff
@@ -8,4 +8,13 @@
 # Hallo Jule & Nicklas, das ist unser Testskript :-) Fuer mehr Smileys hier einmal ein: "JA"
 # :P
 
-# I want a dinosaur please! Preferably a tyrannosaurus
\ No newline at end of file
+# I want a dinosaur please! Preferably a tyrannosaurus
```

Comment:
This one as well.

Reply:
Thanks for the suggestion, but I would like to keep some fun in this script!

Reply:
okay thats fine :D

Reply:
i think that's a great addition to the script, too